libraries/pose/src/custom-posenet.ts

```diff
@@ -87,17 +91,19 @@ const processMetadata = async (metadata: string | Metadata) => {
 
 /**
  * process posenet configuration options
- * @param config a PosenetConfig object
+ * @param config a ModelSettings object
  */
-const fillConfig = (config: Partial<ModelConfig>) => {
-	config = config ? config : {};
+const fillConfig = (config: ModelSettings = {}) => {
```

Comment:
I would just make this `fillConfig = (config: Partial<PoseModelSettings> = {}): PoseModelSettings => {}`
so that it can take a partial but always return a completed object.